app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -7,38 +7,40 @@ import me.hufman.androidautoidrive.cds.CDSVehicleUnits
 import me.hufman.androidautoidrive.phoneui.FlowUtils.addPlainUnit
 import me.hufman.androidautoidrive.phoneui.FlowUtils.format
 import kotlin.math.absoluteValue
+import java.util.*
+import java.time.format.*
 
 class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics) {
 	// general car information
 
 	// RHD is guessed, defaults to LHD
 	// mainly used for driver/passenger window orientation
-	val rightHandDrive = carCapabilities["type_steering"] == "right" ||
+	private val rightHandDrive = carCapabilities["type_steering"] == "right" ||
```

Comment:
Please don't change all of these member visibilities, they have made the PR diff impossible to read.

Reply:
Yes, I see. Will do the cleanup with a later RP.